tests/run-make/thumb-none-qemu/rmake.rs

```diff
@@ -0,0 +1,51 @@
+//! How to run this
+//! $ ./x.py clean
+//! $ ./x.py test --target thumbv6m-none-eabi,thumbv7m-none-eabi tests/run-make
+//!
+//! For supported targets, see `example/.cargo/config`
+
+//@ only-thumb
+
+use std::path::PathBuf;
+
+use run_make_support::{cmd, env_var};
+
+const CRATE: &str = "example";
+
+fn main() {
+    std::env::set_current_dir(CRATE).unwrap();
```

Comment:
Suggestion: this is not needed, the rmake.rs test binary is built and placed in a separate `rmake_out` directory.

Reply:
this line moves into the `tests/run-make/thumb-none-qemu/example` directory. That is important, because that is now the `.cargo/config.toml` takes effect. Maybe there is another way to achieve that, but right now that line is important.

Reply:
Ah right thanks for the correction, I keep forgetting about the .cargo folder